TSC-Charter.md

```diff
@@ -142,18 +114,19 @@ method should be used, such as:
 Multiple-candidate methods may be reduced to simple election by plurality
 when there are only two candidates for one position to be filled. No
 election is required if there is only one candidate and no objections to
-the candidates election. Elections shall be done within the Projects by
-the Collaborators active in the Project.
+the candidates election. Elections shall be done within the projects by
+the Collaborators active in the project.
 
 The TSC will elect from amongst voting TSC members a TSC Chairperson to
-work on building an agenda for TSC meetings and a TSC Director to represent
-the TSC to the Board for a term of one year according to the Node.js Foundation’s
-By-laws. The Chair and Director may be (but are not required to be)
-the same person. The TSC shall hold annual elections to select a TSC
-Chairperson and Director; there are no limits on the number of terms a
-TSC Chairperson or Director may serve.
+work on building an agenda for TSC meetings and a voting OpenJS
```

Comment:
I believe this section may need a bit of clarification or what may be better is to maybe just reference the [appropriate section in the CPC charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members).

As an Impact Project, Node.js is allowed up to 2 voting representatives in the CPC. The way I am reading this section as currently written is that the Node.js project will elect:

> a TSC Chairperson to work on building an agenda for TSC meetings

This person is not mentioned as being a voting member of the CPC

> and a voting OpenJS Cross Project Council (CPC) voting member to represent the TSC in the OpenJS Foundation for a term of one year.

This would make 1 CPC voting member for Node.js. If the Chairperson and the voting representative are the same person, that would still only make 1 representative when Node.js is allowed 2 if the project wants them.

Reply:
I believe the second voting member comes from the community committee

Reply:
Ah! Yes, you are correct. Ignore me :)